### DISCUSSION TITLE
Where can I download the electronic circuit board of PS-CPU_ELECTRONIC_BOARD.PRT in the head of iCub?

### DISCUSSION THREAD

**Discussion by xilihutu888** [robotology/community#491] [Q&A] [answered]
Where can I download the electronic circuit board of PS-CPU_ELECTRONIC_BOARD.PRT in the head of iCub?
Clone all folders but no related electronic circuit boards were found.
Please help me
![image](https://user-images.githubusercontent.com/20624124/111430757-c9960280-8735-11eb-978f-b4101503d57e.png)

**Comment by pattacini** [comment, marked as answer]
This is a duplicate of https://github.com/icub-tech-iit/electronics-boards-public/issues/2.

**Reply by xilihutu888** [reply]
hi, sorry, I didn’t get a reply in the relevant section, so I came to the community to ask a question, do you know the answer? thank you for your reply

**Reply by pattacini** [reply]
Nope @nan2854.
People in the know do work in https://github.com/icub-tech-iit/electronics-boards-public and thus should be already notified.
Just give them enough time to reply.
